src/arche/rules/json_schema.py

```diff
@@ -31,7 +31,11 @@ def check_tags(source_columns, target_columns, tags):
 
     found_tags = list(tags)
     if found_tags:
-        result.add_info(", ".join(found_tags))
+        result.add_info(f"Used - {', '.join(found_tags)}")
```

Comment:
The `not_used_tags` is sorted, but this one is not.
The sorting happens in another place?

Reply:
it's not really needed since list keeps order and I sorted set to get repeated test result, but I sorted it for consistency anyway